### PR TITLE
PRO-1202 users have no drafts

### PR DESCRIPTION
**Pull request by boutell**
Hide "save draft" button, also do not show publication-related labels in manage view, same idea

### CODE REVIEW

File: modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManagerDisplay.vue [hunk 1]
```diff
@@ -119,7 +120,7 @@ export default {
       default() {
         return {};
       }
-    }
+    },
```
Comment (stuartromanek) [inline modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManagerDisplay.vue:123]: extra

**Review by stuartromanek — CHANGES_REQUESTED** [review]
sorry

**Review by stuartromanek — APPROVED** [review]

**Comment by boutell** [comment]
One more time, we're gon' review this PR
Oh yeah, alright, don't stop the review
One more time, we're gon' review this PR
Because of a merge commit

Thanks

File: modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue [hunk 2]
```diff
@@ -180,6 +181,9 @@ export default {
         emoji: '📄'
       };
     },
+    manuallyPublished() {
+      return this.options.localized && !this.options.autopublish;
+    },
```
Comment (stuartromanek) [inline modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue:186]: Should this be part of the publishMixin? the same computed just landed in AposDocsManager

**Review by stuartromanek — COMMENTED** [review]
see above

**Review by stuartromanek — APPROVED** [review]

**Pull request merged**